### PR TITLE
Defect/ffmt-3245: Order pick sheets template breaking in Pickwave

### PR DESCRIPTION
**Pull request by kibo-ajayjangam**

### CODE REVIEW

File: stylesheets/back-office.css [hunk 1]
```diff
@@ -721,4 +721,9 @@ table {
 
   .isSplitPackingClass {
     page-break-after: always;
+}
+ 
+.mz-printable-order-packingslip .slip-product-image {
```
Comment (Kibo-Omkar) [inline stylesheets/back-office.css]: Include pick wave or pick sheet in the class name,
As these classes are used by customers to customize themes as per their needs,
So including feature related names in class helps them to recognize them quickly,
--> Class name suggestion
.mz-printable-order-pickwave-packingslip .pickwave-slip-product-image

**Review by Kibo-Omkar — CHANGES_REQUESTED** [review]
Kindly Resolved the minor review comment.

File: stylesheets/back-office.css (hunk 1 above)
Comment (kibo-ajayjangam) [reply stylesheets/back-office.css]: Resolved

**Review by Kibo-Omkar — APPROVED** [review]

**Review by PapaTaylor — APPROVED** [review]
Looks good

**Pull request merged**